modules/admin_manual/pages/configuration/files/default_files_configuration.adoc

```diff
@@ -20,7 +20,11 @@ image:skeleton-files1.png[image]
 == Additional Configuration
 
 The configuration option `skeletondirectory` available in your `config.php` allows you to configure the directory where the skeleton files are located.
+
 These files will be copied to the data directory of new users.
-Leave empty to not copy any skeleton files.
+
+Leave the `/core/skeleton` directory empty if you do not want to copy any skeleton files.
+
+The value of the `skeletondirectory =>` key **must not be empty** if you decide to use it in your `config.php`.
```

Comment:
`skeletondirectory =>` can be simplified to just `skeletondirectory`.

Reply:
changed